src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
 import * as React from 'react';
 import {Text, View, StyleSheet, TouchableOpacity, Pressable, Platform} from 'react-native';
-import ToggleSwitch from 'toggle-switch-react-native';
 import ToastMessage from 'react-native-toast-message';
 import TextAreaChat from '../../../components/TextAreaChat';
+import ToggleSwitch from '../../../components/ToggleSwitch';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ToggleSwitch` has been changed from an external library to a local file. Ensure that the new component has all the necessary features and behaves as expected.
```diff
- import ToggleSwitch from 'toggle-switch-react-native';
+ import ToggleSwitch from '../../../components/ToggleSwitch';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->